src/common.js

```diff
@@ -94,6 +94,12 @@ export const languages = [
     document: null,
     default_file: "code.rs",
     index: 2,
+    // TODO: remove verbose
+    language_server: ["rust-analyzer", "--verbose"],
+    formatting_options: {
+      ...formatting_options,
+      tabSize: 2,
```

Comment:
We currently use rustfmt's defaults here, which specificies a tab size of 4.
Unless there's a good reason, we should stick with that.